### DISCUSSION TITLE
Error: the pair_coeff of lammps

### DISCUSSION THREAD

**Discussion by Kangxuxin** [deepmodeling/dpgen#1112] [Q&A]
Teacher, the 01.model_devi get a error.

use deepmd-kit at: /public/home/duanxiangmei/softwore/deepmd-kitpair_coeff
ERROR: Incorrect args for pair coefficients (src/input.cpp:1680)
Last command: pair_coeff

I checked the input.lammps file, I found that the pair_coeff parameter was empty.
"pair_style deepmd ../graph.000.pb ../graph.003.pb ../graph.002.pb ../graph.001.pb out_freq ${THERMO_FREQ} out_file model_devi.out
pair_coeff"

If I add ** to pair_coeff in input.lammps in all folders, the 01.model_devi can proceed smoothly. But the second iteration needs to be changed again, which is very troublesome.
I would like to know if this is due to a missing parameter in param.json or machine.json?
Thanks a lot

**Comment by wanghan-iapcm** [comment]
see https://github.com/deepmodeling/dpgen/pull/831

**Reply by Kangxuxin** [reply]
Teacher, I can't find the dpgen/auto_test/template/elastic/lmp/potential.mod file in the dpgen.

**Reply by njzjz** [reply]
#567
